cmd/solizard/solizard.go

```diff
@@ -24,26 +23,29 @@ const Binary = "solizard"
 var (
 	// AbiDir is the directory where all abi files are stored
 	// default is $HOME/solizard/abis
-	AbiDir      = "abis"
+	AbiDir      = "./docs/sample"
```

Comment:
@minias 
Due to security concerns, I prefer using absolute paths. 
While I agree that reading config files based on the execution path of the program provides a convenient user experience.
But the execution path can vary each time, potentially leading to an increase in the number of config files containing critical information, such as private keys, within a single machine.

Therefore, I prefer using absolute paths. However, currently, the directory named `solizard`, which is the same as the binary name, is located in the home directory. 
This can cause inconvenience due to the name collision with the binary. I suggest adding a dot at the beginning, like `.solizard`, to avoid this issue.

Reply:
I agree. I think it is necessary to verify the absolute path because the path may vary from user environment to user.